### PR TITLE
Repair blocklist

### PR DESCRIPTION
**Pull request by missytake**
The check against the blocklist didn't work. Now it does, I tested it.

### CODE REVIEW

File: active_bots/twitterDMs.py [hunk 1]
```diff
@@ -29,6 +29,7 @@ def crawl(self, user):
         :return: reports: (list of report.Report objects)
         """
         reports = []
+        return reports  # TwitterDMs are broken atm
```
Comment (git-sid) [inline active_bots/twitterDMs.py:32]: please move this to another commit/PR. This is not part of the blocklist repair.

**Review by git-sid — CHANGES_REQUESTED** [review]
1 small issue, otherwise LGTM

**Comment by missytake** [comment]
Cool, just merge it if it's LGTM :)

**Review by git-sid — APPROVED** [review]
